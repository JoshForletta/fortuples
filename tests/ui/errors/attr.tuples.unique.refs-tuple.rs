use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::refs_tuple]
    #[tuples::refs_tuple(mut)]
    impl Container<#Tuple> {}
}

fn main() {}
