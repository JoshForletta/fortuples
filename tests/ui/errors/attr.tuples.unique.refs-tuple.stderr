error: `refs_tuple` should be set only once
 --> tests/ui/errors/attr.tuples.unique.refs-tuple.rs:7:15
  |
7 |     #[tuples::refs_tuple(mut)]
  |               ^^^^^^^^^^

error: the first definition was here
 --> tests/ui/errors/attr.tuples.unique.refs-tuple.rs:6:15
  |
6 |     #[tuples::refs_tuple]
  |               ^^^^^^^^^^
